jetstream/statistics.py

```diff
@@ -146,38 +120,73 @@ def __attrs_post_init__(self):
         bigquery.SchemaField("analysis_basis", "STRING"),
     )
 
+    # override the behavior of window_index because this is not
+    # a field in the bigquery schema, and so we need to exclude
+    # it on the Jetstream side
+    window_index: str = Field(default=None, exclude=True)
+
+    class Config:
+        use_enum_values = True
+
+    @validator("ci_width", "point", "lower", "upper", allow_reuse=True)
+    def check_number_fields(cls, v, values, field):
+        if v is not None and not isinstance(v, numbers.Number):
+            if math.isnan(v):
+                return None
+            raise ValueError(f"Expected a number for {field.name}; got {repr(v)}")
+        return v
+
+    @validator("parameter")
+    def normalize_decimal(cls, v):
+        if isinstance(v, Decimal):
+            return str(round(v, 6).normalize())
+        if isinstance(v, float):
+            return str(round(v, 6))
+        return v
+
+    @validator("*")
+    def suppress_infinites(cls, v):
+        if not isinstance(v, float) or math.isfinite(v):
+            return v
+        return None
```

Comment:
These validators were previously in  `StatisticResultCollection`, but they need to be here instead because this is where the actual fields it is validating are. They were also modified slightly because the old version of the schema had `parameter` as a `Decimal` field and I have it as a `float` (if this is a problem let me know and I can change in the schema).

Reply:
I think `float` should be fine